presto-docs/src/main/sphinx/release/release-327.rst

```diff
@@ -0,0 +1,68 @@
+=========================
+Release 327 (16 Dec 2019)
+=========================
+
+General Changes
+---------------
+
+* Fix out-of-order data source reads when using ``work_processor_pipelines``. (:issue:`2144`)
```

Comment:
@sopel39 What is fixed from a user perspective? Why would someone care that reads are out-of-order?

The session property should be `late_materialization`. However, we should probably say just say

> Fix query failure when late materialization is enabled.

Reply:
@electrum yes, from user perspective it was a query failure

Reply:
> Fix query failure when late materialization is enabled.

I would change it to:
```
Fix join query failure when late materialization is enabled. 
```